### ISSUE TITLE
how show text ..

### ISSUE THREAD

**Issue by bandaralsehli** [giandifra/arcore_flutter_plugin#15]
how can add text is there any function for that like function ArCoreNode for shape..
how can improve or add new function to  plugin ......

**Comment by iamprakash13** [comment]
yes that part of arkit(ios only for now). yes that needs to be added.
